core/modules/layout/layout.context.admin.inc

```diff
@@ -1,6 +1,8 @@
 <?php
-module_load_include('inc', 'layout', 'layout.admin');
-
+/**
```

Comment:
Thanks for adding the doc comment, but there should be one blank line before it. (Sorry for being so picky. :wink: )

Reply:
Actually I think it's not standard in this project not to have a space. So this is fine. See examples like admin_bar.module.

Reply:
Ah, OK, then ignore it. :wink: It's just that phpcs nags because of it.

Reply:
I'm not sure why there's no space, but tis the custom. Oh well.